source/newtonian/three_dimensional/ConditionExtensiveUpdater3D.cpp

```diff
@@ -98,7 +98,7 @@ void ConditionExtensiveUpdater3D::operator()(const vector<Conserved3D>& fluxes,
 					" extensive "<<extensives[i].tracers[j]<<std::endl;
 			}
 			face_vec temp = tess.GetCellFaces(i);
-			Conserved3D old_ext(extensives[i]);
+			Conserved3D old_ext;//(extensives[i]);
```

Comment:
Why are you changing the value? it is giving a different result now

Reply:
I reverted it to the previous form